docs/sources/api-response-examples/github/sanitized/repos.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "id":408943403,
+    "node_id":"R_kgDOGF_7Kw",
+    "name":"psoxy",
+    "full_name":"Worklytics/psoxy",
+    "private":false,
+    "owner":{
+      "login":"{\"scope\":\"github\",\"hash\":\"2sH2Fo.Zu09PBKAf5I4pHaoNy5uvcZ3zWs9SiWUBTWQ\"}",
+      "id":"{\"scope\":\"github\",\"hash\":\"VEvLkj8lIGM8HV6s3dhWbUNVGH0v8cK3LYhYxx5DcsE\"}",
+      "node_id":"{\"scope\":\"github\",\"hash\":\"NHcnDQoWyKLZxrrtTxT9AJKyPbhc_0Ju7ynKtCTwI_c\"}",
+      "type":"Organization",
```

Comment:
i think we can in principal skip sanitizing owner where type == Organization, if that's possible.  Or just `owner` on the repos endpoint probably, as in our use case don't expect it to be a user.